source/_docs/upstream-updates.md

```diff
@@ -160,6 +160,14 @@ It's important to relay the need for updating core to maintainers, even if you p
 
 Once you have communicated the issue, you can [manually apply updates from the command line](#apply-upstream-updates-manually-from-the-command-line-and-resolve-merge-conflicts).
 
+### Custom Upstream Updates Not Available
+
+If you know your site's Custom Upstream has updated code, but it's not visible on your Site Dashboard, click on **Check now**:
+
+![Check Now](/source/docs/assets/images/dashboard/check-for-updates.png)
+
+This will trigger a "Code Cache Clear" to verify that the Site Dashboard has fetched the most recent commit. Please note that even after the workflow compeltes, it may take up to a minute before updates appear on the dashboard.

```

Comment:
What does "Code Cache Clear" refer to in this context? If it's a workflow, suggest leaving it out so it's not distracting. Maybe something like: 
> This triggers the Site Dashboard to look for new commits made in the Custom Upstream's remote repository.

Reply:
Or if it's a workflow and we want to keep it, suggest adding supporting context in a tooltip or something